esmvaltool/diag_scripts/mlr/__init__.py

```diff
@@ -7,14 +7,14 @@
 from copy import deepcopy
 from pprint import pformat
 
+import esmvalcore.preprocessor
```

Comment:
whoa you really need the whole of it? :grin:

Reply:
Yes, this script needs a lot of stuff in it :smile: This line was changed by `isort`, so it doesn't matter in the context of this PR.